frontend/packages/console-app/src/components/nav/utils.ts

```diff
@@ -25,12 +25,14 @@ const findIndexForItem = (item: NavExtension, currentItems: NavExtension[]): num
   const after = toArray(insertAfter);
   let count = 0;
   while (count < before.length && index < 0) {
-    index = currentItems.findIndex((i) => i.properties.id === before[count]);
+    const loopCount = count;
```

Comment:
IMO, we should strongly consider refactoring this function. Adding this const puts a bandaid on the lint issue, but the larger issue is that this function is not adhering to functional programming principles. I have some ideas on how we can improve this if you want to talk more about it.